lightning/src/ln/channelmanager.rs

```diff
@@ -2211,20 +2211,20 @@ impl<ChanSigner: ChannelKeys, M: Deref, T: Deref, K: Deref, F: Deref, L: Deref>
 		}
 	}
 
-	fn internal_open_channel(&self, their_node_id: &PublicKey, their_features: InitFeatures, msg: &msgs::OpenChannel) -> Result<(), MsgHandleErrInternal> {
+	fn internal_open_channel(&self, counterparty_node_id: &PublicKey, their_features: InitFeatures, msg: &msgs::OpenChannel) -> Result<(), MsgHandleErrInternal> {
```

Comment:
Personally, for all these methods I'd just call the parameter `node_id` and `features`. From the context, it should be clear that we aren't passing in our own node id and features.

Reply:
I'm ~0 on this. It's quite obvious when you're used to, but for newcomers that's clearly helpful ?

Reply:
Not sure what ~0 means. 😄  Indifferent?

My argument is these are (helpers for) implementing `ChannelMessageHandler`, so it should be clear. I'd be fine with leaving these unchanged and moving to `node_id` and `features` in a follow-up that included updating `ChannelMessageHandler`.

Reply:
Yes indifferent, let's keep it for a follow-up.